examples/features/nse-composition/nse-firewall/kustomization.yaml

```diff
@@ -1,16 +1,5 @@
 ---
```

Comment:
This application actually acts like a firewall, probably it would be better to use at least here patch applied to `apps/nse-firewall-vpp` instead of creating a new file?

Reply:
ok, done